### PR TITLE
[SUBS-807] Apply DLX attributes on queues

### PR DESCRIPTION
**Pull request by ke4**
Also refactor messaging configuration

### CODE REVIEW

File: src/main/java/uk/ac/ebi/subs/validator/aggregator/messaging/AggregatorRoutingKeys.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package uk.ac.ebi.subs.validator.aggregator.messaging;
+
+public class AggregatorRoutingKeys {
+
+    static final String EVENT_VALIDATION_SUCCESS = "validation.success";
```
Comment (fpenim) [inline src/main/java/uk/ac/ebi/subs/validator/aggregator/messaging/AggregatorRoutingKeys.java]: Should this be public?

File: src/main/java/uk/ac/ebi/subs/validator/aggregator/messaging/AggregatorRoutingKeys.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    static final String EVENT_VALIDATION_ERROR = "validation.error";
```
Comment (fpenim) [inline src/main/java/uk/ac/ebi/subs/validator/aggregator/messaging/AggregatorRoutingKeys.java]: Should this be public?

File: src/main/java/uk/ac/ebi/subs/validator/aggregator/messaging/AggregatorRoutingKeys.java (hunk 1 above)
Comment (ke4) [reply src/main/java/uk/ac/ebi/subs/validator/aggregator/messaging/AggregatorRoutingKeys.java]: It is used in the AggregatorMessagingConfiguration, which is in the same package

File: src/main/java/uk/ac/ebi/subs/validator/aggregator/messaging/AggregatorRoutingKeys.java (hunk 2 above)
Comment (ke4) [reply src/main/java/uk/ac/ebi/subs/validator/aggregator/messaging/AggregatorRoutingKeys.java]: It is used in the AggregatorMessagingConfiguration, which is in the same package

**Pull request merged**